packages/block-editor/src/components/block-breadcrumb/index.js

```diff
@@ -0,0 +1,79 @@
+/**
+ * WordPress dependencies
+ */
+import { Button } from '@wordpress/components';
+import { useSelect, useDispatch } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import BlockTitle from '../block-title';
+
+/**
+ * Block breadcrumb component, displaying the hierarchy of the current block selection as a breadcrumb.
+ *
+ * @return {WPElement} Block Breadcrumb.
+ */
+const BlockBreadcrumb = function() {
+	const { selectBlock, clearSelectedBlock } = useDispatch( 'core/block-editor' );
+	const { clientId, parents, hasSelection } = useSelect( ( select ) => {
+		const {
+			getSelectionStart,
+			getSelectedBlockClientId,
+			getBlockParents,
+		} = select( 'core/block-editor' );
+		const selectedBlockClientId = getSelectedBlockClientId();
+		return {
+			parents: getBlockParents( selectedBlockClientId ),
+			clientId: selectedBlockClientId,
+			hasSelection: getSelectionStart().clientId,
+		};
+	}, [] );
+
+	/*
+	 * Disable reason: The `list` ARIA role is redundant but
+	 * Safari+VoiceOver won't announce the list otherwise.
+	 */
+	/* eslint-disable jsx-a11y/no-redundant-roles */
+	return (
+		<nav aria-label={ __( 'Block breadcrumb' ) }>
```

Comment:
Should we just do that and ship? There seem to be disagreement on the best way to do this. We can ship with your proposal and keep discussing this if needed separately.

Reply:
I'm fine with that, someone with the real device could test it to ensure it works universally.

At the moment, `nav` isn't a perfect fit because we use buttons. You mentioned that there is a PR that tries to refactor buttons into link for the Block Navigation. We probably should switch to `nav` once we use links everywhere.  In all tutorials I saw in the past months, the recommend using `nav` with links and menus with buttons 🤷‍♂ 

https://www.w3.org/TR/wai-aria-practices/#breadcrumb

> A breadcrumb trail consists of a list of links to the parent pages of the current page in hierarchical order. It helps users find their place within a website or web application. Breadcrumbs are often placed horizontally before a page’s main content.

However, I don't think we are ready to use links at the moment because we don't change urls so it's all tricky.

Also https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav:

> The HTML `<nav>` element represents a section of a page whose purpose is to provide navigation links, either within the current document or to other documents. Common examples of navigation sections are menus, tables of contents, and indexes.

I'm personally lost in all those recommendations and descriptions 🙃

Reply:
done

Reply:
The downside of using only a list is that it won't be reachable via the VoiceOver rotor (which lists roles like `navigation` and `region`, but not `list`) or by pressing <kbd>R</kbd>/<kbd>D</kbd> on JAWS/NVDA.

They have commands to navigate between lists, but I'm not sure if it's widely used. Most lists on the web don't have labels anyway.

They will see only `Editor footer region` in this case.